Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/burrower.yml

```diff
@@ -31,17 +31,23 @@
     - ActionXenoTailStab
     - ActionXenoAcidNormal
     - ActionXenoOrderConstruction
-    #    - ActionXenoDigTunnel
     - ActionXenoPlantWeeds
-    #    - ActionXenoBurrow
-    #    - ActionXenoTremor
```

Comment:
ActionXenoDigTunnel and ActionXenoBurrow should still be left here commented as notes

Reply:
Ok, think I deleted the comment while trying to debug Construction and didnt put it back